schemas/IDLs/org.gel.models.report.avro/6.0.0/CommonInterpreted.avdl

```diff
@@ -7,6 +7,22 @@ protocol CommonInterpreted {
     import idl "CommonParticipant.avdl";
 
 
+enum SegregationPattern {
```

Comment:
check this out @kevinpetersavage we will need to use this when we move to the new version of the models.
Thanks @Antonior26